tests/framework/session/test_session.py

```diff
@@ -244,6 +244,7 @@ def test_create(
         fake_username,
     ):
         mock_click_ctx = mocker.patch("click.get_current_context").return_value
+        mock_sys_argv = mocker.patch("sys.argv").return_value
```

Comment:
We can make this test a bit more explicit and powerful I think. Something like this (similar changes to the other test in this file):

```suggestion
        mock_sys_argv = mocker.patch("sys.argv", ["kedro", "run", "--params=x"])
```